composer.json

```diff
@@ -4,6 +4,10 @@
     "homepage": "https://www.omise.co/",
     "license": "MIT",
     "require-dev": {
-        "phpunit/phpunit": "^5.7 || ^9.5"
+        "phpunit/phpunit": "^5.7 || ^9.5",
+        "mockery/mockery": "^1.6"
+    },
+    "scripts": {
+        "test": "vendor/bin/phpunit --testdox --colors"
```

Comment:
Should we also add as part of changelog?

Reply:
I don't think it is necessary.